src/popup/router/pages/Names/Claim.vue

```diff
@@ -2,13 +2,13 @@
   <div class="claim">
     <NameListHeader />
     <div class="claim-name-holder">
-      <Input
+      <InputField
         v-model="name"
         :placeholder="$t('pages.names.claim.name-placeholder')"
-        label=".chain"
-        labelPosition="right"
         :error="!validName"
-      />
+      >
+        <span slot="right">{{ `.chain` }}</span>
```

Comment:
```suggestion
        <!-- eslint-disable vue-i18n/no-raw-text -->
        <span slot="right">.chain</span>
```
proposing to drop this rule here: https://github.com/aeternity/superhero-wallet/pull/918